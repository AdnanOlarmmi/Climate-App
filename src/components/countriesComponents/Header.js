import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
  <header className="flex-row">
    <Link title="Home" className="header__home" to="/">&lt;</Link>
    <form className="flex-row">
      <span className="material-symbols-outlined">
        search
      </span>
      <input name="name" type="text" placeholder="country" />
    </form>
    <h4>West African Countries Air Quality Indexes.</h4>
  </header>
);

export default Header;
