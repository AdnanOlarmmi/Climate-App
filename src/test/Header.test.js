import Header from '../components/countriesComponents/Header';


describe('Header', () => {
    it('render Header Component', () => {
      expect(Header).toMatchSnapshot();
    });
  });