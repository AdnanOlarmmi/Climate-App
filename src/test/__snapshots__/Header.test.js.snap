// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render Header Component 1`] = `[Function]`;
